_includes/dbml23/submission.html

```diff
@@ -0,0 +1,44 @@
+<section id="submission" class="services-section section-colored">
+    <div class="container">
+        <div class="row text-left margin-top-2">
+            <div class="col-md-12 text-center">
+                <h2> SUBMISSION GUIDELINES </h2>
+            </div>
+
+            <div class="col-md-12 margin-top-3">
+                <div class="row pr-4 pl-5">
+                    <p>
+                        The workshop will accept both regular papers and short papers (work in progress,
+                        vision/outrageous ideas). All submissions must be prepared in accordance with the IEEE template
+                        available
+                        <a href="https://www.ieee.org/conferences/publishing/templates.html" target="_blank">here</a>.
```

Comment:
This year link to templates looks different. Can you please check ICDE Research track and update?